ros2_example_bazel_installed/ros2_example_apps/BUILD.bazel

```diff
@@ -200,6 +200,80 @@ ros_py_binary(
     ],
 )
 
+ros_py_binary(
+    name = "ap_custom_msg_rosbag",
+    srcs = ["ap_custom_msg_rosbag.py"],
+    data = [
+        # Provide both Python and C++ IDL to see if it can get picked up...
+        ":ros2_example_apps_msgs_py",
```

Comment:
Should work without the py msg lib.

Reply:
https://github.com/RobotLocomotion/drake-ros/pull/317/commits/c7a84ff35f7ac8f64e6263cca6c3a2edf3e7a718